src/Features/CSharp/Portable/CodeFixes/Nullable/CSharpDeclareAsNullableCodeFixProvider.cs

```diff
@@ -190,12 +205,24 @@ static TypeSyntax TryGetSingleTypeArgument(TypeSyntax type)
             }
         }
 
+        private static bool IsExpressionSupported(SyntaxNode node)
+        {
+            return node.IsKind(
+                SyntaxKind.NullLiteralExpression,
+                SyntaxKind.AsExpression,
+                SyntaxKind.DefaultExpression,
+                SyntaxKind.DefaultLiteralExpression,
+                SyntaxKind.ConditionalExpression,
+                SyntaxKind.ConditionalExpression,
```

Comment:
`SyntaxKind.ConditionalExpression` is here twice

Reply:
Thanks for spotting this. Removed.